src/modules/core/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -11,12 +11,16 @@ interface Appearance {
 interface Props extends HTMLAttributes<HTMLDivElement> {
   /** Appearance object */
   appearance?: Appearance;
+  className?: string;
 }
 
 const displayName = 'DropdownMenu';
 
-const DropdownMenu = ({ appearance, children, ...props }: Props) => (
-  <div className={getMainClasses(appearance, styles)} {...props}>
+const DropdownMenu = ({ appearance, className, children, ...props }: Props) => (
+  <div
+    className={`${getMainClasses(appearance, styles)} ${className}`}
```

Comment:
This is not a valid change.

I know why you did it, to able to style the component directly, without having to deal with resetting the various styles applied by themes.

But the way this was set up is the standard way most core components styles are set up. So please revert this, and just deal with resetting the styles where you need them